src/main/default/lwc/flowProgressIndicator/__tests__/flowProgressIndicator.test.js

```diff
@@ -0,0 +1,187 @@
+import { createElement } from 'lwc';
+import FlowProgressIndicator from 'c/flowProgressIndicator';
+
+function createTestComponent() {
+  const element = createElement('c-flow-progress-indicator', {
+    is: FlowProgressIndicator
+  });
+  document.body.appendChild(element);
+
+  return element;
+}
+
+// Helper function to wait until the microtask queue is empty.
+function allPromisesResolution() {
+  // eslint-disable-next-line no-undef
+  return new Promise((resolve) => setImmediate(resolve));
+}
+
+jest.mock('@salesforce/label/c.cookbook_LogIn', () => ({ default: 'Log in' }), {
+  virtual: true
+});
+jest.mock(
+  '@salesforce/label/c.cookbook_DescribeProblem',
+  () => ({ default: 'Describe problem' }),
+  {
+    virtual: true
+  }
+);
+jest.mock(
+  '@salesforce/label/c.cookbook_ProvideDetails',
+  () => ({ default: 'Provide details' }),
+  {
+    virtual: true
+  }
+);
+jest.mock(
+  '@salesforce/label/c.cookbook_ReviewHelp',
+  () => ({ default: 'Review help' }),
+  {
+    virtual: true
+  }
+);
```

Comment:
```suggestion
const mockLabels = [
    {
        label: '@salesforce/label/c.cookbook_DescribeProblem',
        default: 'Describe problem',
    },
    {
        label: '@salesforce/label/c.cookbook_ProvideDetails',
        default: 'Provide details',
    },
    {
        label: '@salesforce/label/c.cookbook_ReviewHelp',
        default: 'Describe problem',
    },
];
mockLabels.forEach(label => {
    jest.mock(
        label.label,
        () => ({ default: label.default }),
        { virtual: true }
    );
});
```

What do you think of that?

Reply:
But apparently Jest don't like when we use variables inside mocks, look at this:
<img width="1764" alt="jest_error" src="https://user-images.githubusercontent.com/86681870/151173103-7db2e035-6ef8-4393-9043-189b08d3485b.png">